### ISSUE TITLE
UI: Landing page 

### ISSUE THREAD

**Issue by dayanadithyan** [dca123/dayan-website#3]
Change boxes to uniform size (x3) 

1) My writing
2) My coding
3) My work

[Square boxes] 

<img width="1230" alt="image" src="https://github.com/dca123/dayan-website/assets/65873585/b7d32d72-9f40-4b32-b7c5-a29dc56568ab">

**Comment by dayanadithyan** [comment]
Proposed backgroup shades: 
Two:

Light shade = 32,34,34 RGB
Dark shade = 25,26,26 RGB

Split from box issue (#2)

**Comment by dayanadithyan** [comment]
Changes look good, to close out, content:

Box 1: 
My work:
Select written pieces on my applied data-science projects
[2 snippets]

Box 2:
My writing:
Personal musings on everything from TV to video-games and life
[2 snippets]

Box 3:
My code:
Select data-science projects relevant to academia
[2 snippets]

About me: (With line spacing)

I like symbols, both words and numbers. My research in language is driven by this curiosity in how language shapes human experience. 

I earned a Bachelor’s degree in Liberal Arts & Sciences from Quest University (British Columbia), with a focus on mathematics and critical theory.

Currently, I serve as a Senior Researcher in NLP R&D,(and implementation,) and TMT policy. Before that, I consulted at McKinsey for 2 years. Both focused on the APAC.
